src/main/kotlin/io/openfuture/state/controller/domain/request/SaveWalletRequest.kt

```diff
@@ -0,0 +1,6 @@
+package io.openfuture.state.controller.domain.request
```

Comment:
The same for WalletDto:
1. Move to the Controller definition
1. Nullable properties + javax.valiation annotations